mycroft/skills/skill_updater.py

```diff
@@ -14,10 +14,10 @@
 #
 """Periodically run by skill manager to update skills and post the manifest."""
 import os
+import tempfile
 import sys
 from datetime import datetime
 from time import time
-
```

Comment:
This space should remain here I think since above are builtin packages and msm is a non-system package but not included in the mycroft module